easybuild/easyconfigs/m/Molpro/Molpro-mpp-2010.1.23-Linux_x86_64.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Molpro'
+easyblock = 'EB_Molpro'
+versionprefix = 'mpp-'
+version = '2010.1.23'
+versionsuffix = '-Linux_x86_64'
+
+homepage = 'https://molpro.net'
+description = """Molpro is a complete system of ab initio programs for molecular electronic structure calculations."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+#toolchain = {'name': 'intel', 'version': '2015a'}
+#toolchainopts = {'usempi': True}
+
+# no source URL available, requires registration to download
+sources = ['%(namelower)s-%(versionprefix)s%(version)s%(versionsuffix)s.sh']
+patches = ['%(namelower)s-%(versionprefix)s%(version)s%(versionsuffix)s.patch']
+precompiled_binaries = True
+
+# license file
+#import os
+#license_file = os.path.join(os.getenv('HOME'), 'licenses', name, 'license.lic')
```

Comment:
If it's to be installed at a site that has a licence token, it might be helpful for them to see how the licence file can be supplied.